packages/react-components/react-nav-preview/library/src/components/AppItem/useAppItemStyles.styles.ts

```diff
@@ -1,22 +1,35 @@
 import { makeStyles, mergeClasses } from '@griffel/react';
 import type { SlotClassNames } from '@fluentui/react-utilities';
 import type { AppItemSlots, AppItemState } from './AppItem.types';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
+import { useIconStyles, useRootDefaultClassName } from '../sharedNavStyles.styles';
 
 export const appItemClassNames: SlotClassNames<AppItemSlots> = {
   root: 'fui-AppItem',
-  // TODO: add class names for all slots on AppItemSlots.
-  // Should be of the form `<slotName>: 'fui-AppItem__<slotName>`
+  icon: 'fui-AppItem__icon',
 };
 
 /**
  * Styles for the root slot
  */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    marginInline: '4px',
```

Comment:
Should 'as' button still have a cursor: pointer applied?

Reply:
ah, good call. Seems this issue exists on all nav items, so I'll make the change there.